Makefile

```diff
@@ -125,17 +129,9 @@ deploy-helm: manifests
 
 undeploy: kustomize
 	${KUSTOMIZE} build config/default | kubectl delete -f -
-	kubectl delete validatingwebhookconfigurations.admissionregistration.k8s.io inferenceservice.serving.kserve.io
```

Comment:
Why these are deleted ?

Reply:
The webhooks are already deleted by the first command. So there is no need to delete it separately.
![Screenshot from 2023-07-31 11-20-24](https://github.com/kserve/kserve/assets/90966311/cc16ab54-b742-4061-b172-81a26a9884f3)